english/dream/fly/asia/japan/japan.md

```diff
@@ -2,4 +2,6 @@ You have decided to leave India and fly to Japan.
 
 You land in Tokyo Haneda Airport in the morning.
 
-What is your first stop in Japan?
\ No newline at end of file
+What is your first stop in Japan?
+
+[Board a train](english/dream/fly/asia/japan/train.md) heading to Mount Fuji.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.